test/aws/rails/notifications_instrument_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+module Aws
+  module Rails
+    describe 'NotificationsInstrument Plugin' do
+      let(:client) do
+        Client = Aws::SES::Client
+        Client.add_plugin(Aws::Rails::Notifications)
+        Client.new(stub_responses: true, logger: nil)
+      end
+
+      it 'adds instrumentation on each call' do
+        out = {}
+        ActiveSupport::Notifications.subscribe(/aws/) do |name, start, finish, id, payload|
+          out[:name] = name
+          out[:payload] = payload
+        end
+        client.get_send_quota
```

Comment:
Either I'd make a base client (that's not a real client) and use a test operation, or, use SES like you have here but use the send_raw_email operation (because we depend on it anyway)

Reply:
Wanted a method that didnt require parameters.  But changed.